test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -170,7 +170,7 @@ public void testActiveClusterReset() throws Exception {
                 TableOptions.fromProtoSchema(LogReplicationMetadata.ReplicationStatusVal.class));
 
         // Wait the polling period time before verifying sync status (to make sure it was updated)
-        Sleep.sleepUninterruptibly(Duration.ofSeconds(LogReplicationAckReader.ACKED_TS_READ_INTERVAL_SECONDS + 1));
+        Sleep.sleepUninterruptibly(Duration.ofSeconds(LogReplicationAckReader.ACKED_TS_READ_INTERVAL_SECONDS + delta));
```

Comment:
any reason for increasing the sleep duration?

Reply:
@Jas0nch had added it probably to avoid some tests failing. Reverting now.